stylesheets/styles.css

```diff
@@ -0,0 +1,30 @@
+/* Include default AsciiDoc styles here... */
+@import "https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,400,400italic,600,600italic%7CNoto+Serif:400,400italic,700,700italic%7CDroid+Sans+Mono:400,700";
+@import "https://cdn.jsdelivr.net/gh/asciidoctor/asciidoctor@2.0/data/stylesheets/asciidoctor-default.css";
```

Comment:
Should probably include this default stylesheet explicitly instead of depending on asciidoc's CDN if we are going to keep it.